src/Parser.ts

```diff
@@ -96,10 +96,10 @@ export class _Parser {
         }
         case 'text': {
           let textToken = token;
-          let body = this.renderer.text(textToken);
+          let body = this.renderer.text(textToken) as string;
```

Comment:
This `as string` seems like the types are wrong. Is there a way to improve it so we don't need the assertion 🤔

Reply:
This is needed because the renderer is generic so at this point `this.renderer.tedt()` could return any type. We are assuming that the RendererOutput applies to all renderer functions so the user will have to change every function (including this one) if they want to return something other than a string. The correct way to type this would be to have a generic param for every renderer function. I feel like that is a bit overkill.